packages/sanity/src/core/i18n/bundles/studio.ts

```diff
@@ -68,6 +68,310 @@ export const studioLocaleStrings = {
+  /** Label for area section that indicates that you can choose your workspace */
+  'navbar.workspace-menu.choose-your-workspace-label': 'Choose your workspace',
+
+  /** Label for action to choose a different workspace */
+  'navbar.workspace-menu.action.choose-another-workspace': 'Choose another workspace',
+
+  /** Label for action to add a workspace */
+  'navbar.workspace-menu.action.add-workspace': 'Add workspace',
+
+  /** --- New Document --- */
+
+  /** Placeholder for the "search" input within the new document menu */
+  'navbar.new-document.search': 'Search',
+
+  /** Loading indicator text within the new document menu */
+  'navbar.new-document.loading': 'Loading…',
+
+  /** Title for "Create new document" Also used for accessibility */
+  'navbar.new-document.title': 'Create new document',
+
+  /** Label for when no document types are found when searching them in the new document menu */
+  'navbar.new-document.no-document-types': 'No document types',
+
+  /** Label for "Create new document" action */
+  'navbar.new-document.action.create-new-document': 'New document...',
+
+  /** Message for when no results are found for a specific search query in the new document menu */
+  'navbar.new-document.no-results': 'No results for',
+
+  /** Message for when there are no document options in the new document menu */
+  'navbar.new-document.no-documents-found': 'No documents found',
+
+  /** Accessibility label for options in the new document menu */
+  'nav-bar.new-document.new-document': 'New document',
+
+  /** Error label for when a user is unable to create a document */
+  'nav-bar.new-document.error.unable-to-create-document': 'create this document',
+
+  /** --- Search --- */
+
+  /** Accessibility label for Search Input for when there are no recent searches */
+  'navbar.search.search-results-label': 'Search results',
+
+  /** Accessibility label for Search Input for when there are recent searches */
+  'navbar.search.recent-searches-label': 'Recent searches',
+
+  /** Action label for clearing search filters */
+  'navbar.search.action.clear-filters': 'Clear filters',
+
+  /** Accessibility label for filtering by document type */
+  'navbar.search.action.filter-by-document-type': 'Filter by document type',
+
+  /** Label for placeholder for Filter (singular) */
+  'navbar.search.filter-label_one': 'Filter',
+
+  /** Label for placeholder for Filters (plural) */
+  'navbar.search.filter-label_other': 'Filters',
+
+  /** Accessibility label for command list for Document types */
+  'navbar.search.document-types-label': 'Document types',
+
+  /** Label for when no documents are found for a filter are found */
+  'navbar.search.no-matches-found': `No matches for {{filter}}`,
+
+  /** Accessibility action label for clearing document type filters */
+  'navbar.search.action.clear-type-filters': 'Clear checked filters',
+
+  /** Clear action label for clearing document type filters */
+  'navbar.search.action.clear-type-label': 'Clear',
```

Comment:
I thought about this for a while and I considered it, the reason why I didn't go with it was: There are places where the labels are used both in accessbility instances AND in the interface itself. Which means, to me, that it's probably preferrable that it's in the comments to those keys explicitly said when it's also used the an accessibility setting vs have a separate accessibility key for it.

I don't have a strong opinion on this, however, I was trying to avoid a situation of possible confusion because I've come across at least a handful of cases where this has happened (the situation I pointed above)

cc @rexxars